core/src/font.rs

```diff
@@ -698,7 +733,9 @@ impl<'gc> Font<'gc> {
         let mut x = Twips::ZERO;
         while let Some((pos, c)) = char_indices.next() {
             let c = c.unwrap_or(char::REPLACEMENT_CHARACTER);
-            if let Some(glyph) = self.get_glyph_for_char(c) {
+            if let Some(render_data) = self.get_glyph_render_data(c) {
```

Comment:
This looks wrong: `self` is a `Font` here, not a `FontSet`, so there's no possibility to do per-character fallback.

Reply:
But this logic is only present on `FontSet`, and we don't have a `FontSet` to use inside `Font::evaluate`?